newIDE/app/public/external/utils/path-editor.js

```diff
@@ -93,9 +99,9 @@ const render = headerObject => {
   ); // Don't allow the user to enter any characters that would lead to an invalid path
   const state = headerObject.state;
   state.name = headerObject.nameInput.value;
-  state.baseExportPath = state.folderPath + '/' + state.name;
-  state.fullPath = state.folderPath + '/' + state.name + state.extension;
-  headerObject.saveFolderLabel.textContent = state.folderPath + '/';
+  state.baseExportPath = state.folderPath + '\\' + state.name;
```

Comment:
This won't work on Windows, better keep "/"

Reply:
sorry, I dont know why I thought that electron would magically sort that :) Its fixed now